projects/hslayers/src/css/app.scss

```diff
@@ -156,6 +156,10 @@ $mobile-layout-breakpoint: $mobile-layout-breakpoint-value +'px' !default; //thi
 		font-size: x-large !important;
 	}
 
+	.cursor-pointer {
```

Comment:
Wow, when did this go out? I thought we have that class present in SCSS for the whole time 😮